docs/modules/haplocheck.md

```diff
@@ -0,0 +1,14 @@
+---
+Name: Haplocheck
+URL: https://github.com/genepi/haplocheck
+Description: >
```

Comment:
```suggestion
name: Haplocheck
url: https://github.com/genepi/haplocheck
description: >
```

The CI is [failing](https://github.com/ewels/MultiQC/actions/runs/6048562796/job/16414157576?pr=1876) to validate this block.

That's because the keys need to be [lowercased](https://multiqc.info/docs/development/modules/#documentation).